nl.js

```diff
@@ -58,7 +58,7 @@ generic.password Wachtwoord Invoeren
 generic.register Registreren
 generic.login Inloggen
 generic.accuracy Nauwkeurigheid
-player.waiting wachten op spelers
+player.waiting Wachten op spelers
```

Comment:
hoeft niet per se een hoofdletter te hebben

Reply:
ill fix it post merge

Reply:
fixed: https://github.com/mty22/krunker.io-translations/commit/b08d59bfb12a7f6a3fb7e6d37b36bcbf490e92dc